handler/src/main/java/io/netty/handler/ssl/JdkSslContext.java

```diff
@@ -160,7 +160,37 @@ public JdkSslContext(SSLContext sslContext, boolean isClient,
     public JdkSslContext(SSLContext sslContext, boolean isClient, Iterable<String> ciphers,
```

Comment:
@slandelle maybe mark this constructor as `@deprecated` ?

Reply:
sure